setup.py

```diff
@@ -189,6 +189,7 @@ def write_version(filename=os.path.join(*['airflow',
     'google-auth>=1.0.0, <2.0.0dev',
     'google-auth-httplib2>=0.0.1',
     'google-cloud-container>=0.1.1',
+    'google-cloud-bigtable==0.31.0',
```

Comment:
Can we use a version range instead of pinning to an exact version?

Reply:
The problem is that those libraries are pretty unstable and relying on automated upgrade for future versions is pretty bad idea - last time when @DariuszAniszewski  upgraded 0.30.0 (I think) -> 0.31.0 it actually broke the operator and some fixes had to be made (and we cannot really rely on future versions to not break anything). Other dependencies also made breaking changes quite recently when they automatically upgraded (infamous flask-appbuilder for example). There is a whole discussion on keeping fixed vs. upper-open version number on the devlist (I am involved but had no time to take a closer look at this yet). I think for now - until general approach is implemented, I think it's safer to keep it fixed.